help/styles/styles.css

```diff
@@ -361,6 +427,22 @@ main .section:empty {
     margin: 32px 24px;
     padding: 40px;
   }
+
+  .article-short .default-content-wrapper h2 {
+    margin-bottom: 32px;
+    font-size: var(--heading-font-size-m);
+    line-height: 3rem;
+  }
+
+  .article-long .default-content-wrapper h2 {
+    font-size: var(--heading-font-size-l);
```

Comment:
font-size defined at line 349 already, could be removed from here.

Reply:
removed, thanks for noticing!